### PR TITLE
[fluentd-gcp addon] Pass region in seperate field

### PR DESCRIPTION
**Pull request by serathius**
This PR makes location passed to event-exporter based on `MULTIZONE` env.

Fixes https://github.com/kubernetes/kubernetes/issues/62399
```release-note
NONE
```
/cc @loburm

### CODE REVIEW

**Comment by serathius** [comment]
@gmarek can you approve?

File: cluster/gce/util.sh [hunk 1]
```diff
@@ -984,6 +984,7 @@ ETCD_CA_CERT: $(yaml-quote ${ETCD_CA_CERT_BASE64:-})
 ETCD_PEER_KEY: $(yaml-quote ${ETCD_PEER_KEY_BASE64:-})
 ETCD_PEER_CERT: $(yaml-quote ${ETCD_PEER_CERT_BASE64:-})
 ENCRYPTION_PROVIDER_CONFIG: $(yaml-quote ${ENCRYPTION_PROVIDER_CONFIG:-})
+REGION: $(yaml-quote ${REGION})
```
Comment (wojtek-t) [inline cluster/gce/util.sh]: please put it next to ZONE (line 892)

File: cluster/gce/gci/configure-helper.sh [hunk 2]
```diff
@@ -2112,7 +2112,12 @@ function update-prometheus-to-sd-parameters {
 
 # Updates parameters in yaml file for event-exporter configuration
 function update-event-exporter {
-    sed -i -e "s@{{ *event_exporter_zone *}}@${ZONE:-}@g" "$1"
+  if [[ "${MULTIZONE:-}" == "true" ]]; then
```
Comment (wojtek-t) [inline cluster/gce/gci/configure-helper.sh]: It's not obvious to me this condition is actually correct.

What you're trying to achieve?
Is the purpose of that handle regional clusters? Not every multizonal one is a regional one...

Comment (serathius) [reply cluster/gce/gci/configure-helper.sh]: I was shadowing fix for regional clusters https://github.com/kubernetes/kubernetes/commit/81c6bb6ec2e1806d67cdde96ef49781271e9fee9 by @loburm 

We need to rethink idea of location of kubernetes cluster and introduce it not only for stackdriver.

Comment (wojtek-t) [reply cluster/gce/gci/configure-helper.sh]: The test fix isn't really correct - you can take a look into:
https://github.com/kubernetes/kubernetes/pull/61892/files
to a better way of fixing that.

> We need to rethink idea of location of kubernetes cluster and introduce it not only for stackdriver.

Yes and no. In majority of cases we know exactly if we want zone or region (it's the case for pretty much all gce-related stuff). 
We need a good story for SD (and maybe couple more?, but I don't think it affects many things).

Also, there is huge difference between fixing tests and fixing production code. We can't afford breaking production code for multizonal clusters. And this PR is clearly changing that behavior.
I would like to be sure that we won't break any of those...

File: cluster/gce/util.sh (hunk 1 above)
Comment (serathius) [reply cluster/gce/util.sh]: done

File: cluster/gce/gci/configure-helper.sh (hunk 2 above)
Comment (serathius) [reply cluster/gce/gci/configure-helper.sh]: I changed if so it depends on MULTIMASTER value.

**Pull request merged**